i18n/zh-tw.toml

```diff
@@ -46,21 +46,21 @@ other = "創建文檔議題"
 [post_create_project_issue]
 other = "創建項目議題"
 [post_posts_in]
-other = "張貼在"
+other = "張貼於"
 [post_reading_time]
-other = "分鐘閱讀"
+other = " 分鐘左右可讀完"
```

Comment:
Based on my research, the original prompt is better here, please revert this change or explain why your new text is better.